api/python/cellxgene_census/tests/README.md

```diff
@@ -45,17 +45,202 @@ When run, please record the results in this file (below) and commit the change t
 - date
 - config:
   - EC2 instance type and any system config (i.e., swap)
-  - host and OS as reported by `uname -a`
+  - host and OS as reported by `uname -a`. **Please remove IP address**
   - Python & package versions and OS - suggest capturing the output of `tiledbsoma.show_package_versions()`
   - The Census version used for the test (i.e., the version aliased as `latest`). This can be easily captured using `cellxgene_census.get_census_version_description('latest')`
   - the cellxgene_census package version (ie., `cellxgene_census.__version__`)
 - any run notes
 - full output of: `pytest -v --durations=0 --expensive ./api/python/cellxgene_census/tests/`
 
+## 2023-07-26
+
+- Host: EC2 instance type: `r6id.x32xlarge`, all nvme mounted as swap.
```

Comment:
typo:  should be r6id.32xlarge

(i.e., only a single `x`)

Reply:
fixed